notebooks/02-ingestion/15-native-dimensions.ipynb

```diff
@@ -0,0 +1,786 @@
+    "\n",
+    "ingestionSpec = {\n",
+    "  \"type\": \"kafka\",\n",
+    "  \"spec\": {\n",
+    "    \"ioConfig\": ioConfig,\n",
+    "    \"tuningConfig\": tuningConfig,\n",
+    "    \"dataSchema\": dataSchema\n",
+    "  }\n",
+    "}\n",
+    "\n",
+    "print(json.dumps(ingestionSpec, indent=5))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "20470873-bf5b-48ee-8842-22ccdcd084af",
+   "metadata": {},
+   "source": [
+    "Submit the revised specification for this table to Druid by running the next cell."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "84e52d35-6f29-4747-b706-4a16672160da",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "supervisor = requests.post(f\"{druid_host}/druid/indexer/v1/supervisor\", json.dumps(ingestionSpec), headers=generalHeaders)\n",
+    "print(supervisor.status_code)\n",
+    "druid.sql.wait_until_ready(table_name, verify_load_status=False)\n",
+    "print(\"Ready to go!\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5ea350ef-5c77-4cc4-ae38-7a8b61e5679f",
+   "metadata": {},
+   "source": [
+    "Review the schema for the table by running the next cell."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "51b18176-9e11-415c-ad97-5ce12db71713",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "display.table(table_name)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4fc8f87b-0b06-4061-8bd2-1f5e7a02f480",
+   "metadata": {},
+   "source": [
+    "Notice that `views`, `upvotes`, and `comments` have been detected as a BIGINT.\n",
+    "\n",
+    "Stop ingestion and drop the table by running the next cell."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "5e4daaa9-3bc5-40f8-a4d5-869a1e73f92f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print(f'Pause streaming ingestion: [{requests.post(f\"{druid_host}/druid/indexer/v1/supervisor/{table_name}/suspend\",\"\")}]')\n",
+    "print(f'Shutting down running tasks ...')\n",
+    "\n",
+    "tasks = druid.tasks.tasks(state='running', table=table_name)\n",
+    "while len(tasks)>0:\n",
+    "    for task in tasks:\n",
+    "        print(f\"...stopping task [{task['id']}]\")\n",
+    "        druid.tasks.shut_down_task(task['id'])\n",
+    "    tasks = druid.tasks.tasks(state='running', table=table_name)\n",
+    "        \n",
+    "print(f'Reset offsets for re-runnability: [{requests.post(f\"{druid_host}/druid/indexer/v1/supervisor/{table_name}/reset\",\"\")}]')\n",
+    "print(f'Terminate streaming ingestion: [{requests.post(f\"{druid_host}/druid/indexer/v1/supervisor/{table_name}/terminate\",\"\")}]')\n",
+    "print(f\"Drop datasource: [{druid.datasources.drop(table_name)}]\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "44738d6d-cec2-40ad-aaba-998c758c63f4",
+   "metadata": {},
+   "source": [
+    "## Clean up\n",
+    "\n",
+    "Run the following cell to stop the data generator."
```

Comment:
I recommend picking one type of intro and sticking with it (if possible). 
So either "Run the next cell to..." or "...by running the next cell."